src/variables/selectors/index.tsx

```diff
@@ -210,7 +211,22 @@ export const getVariable = (state: AppState, variableID: string): Variable => {
   }
 
   if (!vari.selected.length && vals.length) {
-    vari.selected.push(vals[0])
+    try {
+      vari.selected.push(vals[0])
+    } catch (err) {
+      // Temporary measure to resolve errors relating to pushing into non-extensible object.
+      reportErrorThroughHoneyBadger(err, {
+        name: 'Failed to set selected variable to default, zero-indexed value',
+        context: {
+          variable: vari,
+          variableType: vari.arguments.type,
+          normalizedVariable: vals,
+          variableLenth: vari.selected.length,
+          normalizedVariableLength: vals.length,
+          appState: state,
```

Comment:
I think maybe instead of sending the whole app state, we should start with just the variables state.

Reply:
Good point. What if we send something slightly broader than `variables`, just to make sure we have all of the relevant information?

1. `state.resources` - One level higher than `variables` in state, so that we can also see `buckets` `dashboards` and `cells`
2. `state.identity` - so that we know which user @ influxdata hit the error, so we can ask for any context.